Probesys/bileto
# new: Add a page to display information about a user

## Related issue(s)



https://github.com/Probesys/bileto/issues/582

## How to test manually



1. Go to the admin > users
2. Verify that you can click on users, and that a badge appears on the right, displaying the number of authorizations per users
3. Click on a user
4. Verify that you can see information about the user
5. Verify you can grant / revoke authorizations from this page
6. Verify you can go to the edit form from the "Actions" menu

## Checklist
